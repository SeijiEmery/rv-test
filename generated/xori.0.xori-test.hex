0000000    13  c2  01  00  93  c2  11  00  93  c3  a1  41  13  c4  a1  41
0000020    93  c4  02  00  13  00  00  00  13  00  00  00  13  00  00  00
0000040    13  00  00  00  13  00  00  00  73  00  10  00                
0000054
