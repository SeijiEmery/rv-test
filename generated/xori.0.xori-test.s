xori    x4, x3, 0
xori    x5, x3, 1

xori    x7, x3, 1050
xori 	x8, x3, 1050
xori	x9, x5, 0
ebreak
