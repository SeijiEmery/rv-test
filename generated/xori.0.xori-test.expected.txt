tests/xori.test.s:1
R3 = 0
R4 = 0
R5 = 1
R7 = 1050
R9 = 1
